cumulusci/cli/cci.py

```diff
@@ -115,9 +115,7 @@ def check_latest_version():
         click.echo("Checking the version!")
         if result:
             click.echo(
-                """An update to CumulusCI is available. Use {} to update.""".format(
-                    get_cci_upgrade_command()
-                )
+                f"""An update to CumulusCI is available. Use {get_cci_upgrade_command()} to update."""
```

Comment:
```suggestion
                f"""An update to CumulusCI is available. To install the update, run this command: {get_cci_upgrade_command()}"""
```

(This is not related to f-strings, but it's a suggestion that Melissa made the other day.)